cognitive/src/main/scala/com/microsoft/azure/synapse/ml/geospatial/AzureMapsHelpers.scala

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.geospatial
+
+import com.microsoft.azure.synapse.ml.build.BuildInfo
+import com.microsoft.azure.synapse.ml.cognitive.{HasAsyncReply, HasServiceParams, HasSubscriptionKey, URLEncodingUtils}
+import com.microsoft.azure.synapse.ml.io.http.{CustomInputParser, HTTPInputParser, HasURL}
+import com.microsoft.azure.synapse.ml.io.http._
+import com.microsoft.azure.synapse.ml.io.http.HandlingUtils._
+import org.apache.http.client.methods.{HttpGet, HttpPost, HttpRequestBase}
+import org.apache.http.entity.{AbstractHttpEntity, StringEntity}
+import org.apache.http.impl.client.CloseableHttpClient
+import org.apache.spark.ml.param._
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.StructType
+
+import java.net.{URI, URLEncoder}
+import java.util.concurrent.TimeoutException
+import scala.concurrent.blocking
+import scala.language.{existentials, postfixOps}
+import spray.json.DefaultJsonProtocol.{StringJsonFormat, seqFormat}
+
+trait HasAddressInput extends HasServiceParams with HasSubscriptionKey with HasURL {
```

Comment:
Will this ne needed if theres no more non-batch endpoint

Reply:
Cleaned it up